File: C#/Lv_0/문자열 정렬하기(1).cs
using System;
using System.Text.RegularExpressions;

public class 문자열_정렬하기1 {
    public int[] solution(string my_string) {
        string num = Regex.Replace(my_string, @"\D", "");
        int[] answer = new int[num.Length];
        for(int i=0; i<num.Length; i++)
            answer[i] = (int)num[i] - 48;
        Array.Sort(answer);
        return answer;
    }
}

File: JavaScript/Lv_0/문자열 정렬하기(1).js
function solution(my_string) {
    var answer = [];
    my_string = my_string.replace(/[^0-9]/g, "").split("");
    for(let i of my_string) answer.push(Number(i));
    return answer.sort((a,b) => a-b);
}

// 더 요약한 방법
// function solution(my_string) {
//     return my_string.match(/\d/g).sort((a, b) => a - b).map(n => Number(n));
// }

// 또 다른 방법
// function solution(my_string) {
//     return my_string.replace(/[^\d]/g,'').split('').map(v=>+v).sort();
// }